src/App.tsx

```diff
@@ -1,15 +1,32 @@
 import "@mantine/core/styles.css";
-import { Center, Code, MantineProvider, Blockquote } from "@mantine/core";
+import {
+  Center,
+  Code,
+  MantineProvider,
+  Blockquote,
+  AppShell,
+} from "@mantine/core";
 import { theme } from "./theme";
+import Navbar from "/Users/jas/school/gatech/hytech/query-frontend/src/Navbar.tsx";
```

Comment:
don't use absolute paths. 

replacing "/Users/jas/..." with "./Navbar.tsx" should work